src/Core/Domain/Product/Combination/QueryResult/CombinationDetails.php

```diff
@@ -66,22 +66,22 @@ class CombinationDetails
     private $impactOnWeight;
 
     /**
-     * @param string $ean13
+     * @param string $gtin comment here explain rename
      * @param string $isbn
      * @param string $mpn
      * @param string $reference
      * @param string $upc
      * @param DecimalNumber $impactOnWeight
      */
     public function __construct(
-        string $ean13,
+        string $gtin,
```

Comment:
This is also a BC break, with PHP8 you can have named arguments, therefore renaming arguments is a BC.

Reply:
This is not part of our backward compatibility promise so far https://github.com/PrestaShop/ADR/blob/master/0017-backward-compatibility-promise.md

But you're right we should probably update the ADR for the V9

Reply:
The BC is acceptable here in favor of the consistency, but it should be detailed in the PR description for future doc update